jenkins-apb/roles/provision-jenkins-apb/tasks/main.yml

```diff
@@ -0,0 +1,178 @@
+---
+# ----------------------------
+# Jenkins APB
+# ----------------------------
+# This role uses the OpenShift template that is included as the basis
+# for the tasks listed below.  It includes both ephemeral and persistent
+# versions.  It also includes the ability to use source-to-image to customize
+# the Jenkins instance with jobs and plugins.
+# See https://github.com/openshift/jenkins#installing-using-s2i-build for more information.
+#
+# There are a number of workarounds required which are noted in comments
+# near the action.
+# ----------------------------
+- name: Create Namespace
```

Comment:
This is no longer needed as the Ansible Service Broker creates the namespace for you.